### DISCUSSION TITLE
Code Text kirim Enter Baris

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#22] [Tanya Jawab] [answered]
hi rio,

saya mau tanya cara menuliskan supaya bisa enter ke bawah / pindah baris di wa melalui codingan di PHP dalam mengirim pesan.
saya sudah coba br , \n tidak bisa.

Apakah bisa ?

**Comment by rioastamal** [comment, marked as answer]
Tentu bisa. Karakter `\n` harus didalam string dengan double quote `"..."` agar bisa diinterpretasi sebagai baris baru di PHP.

```php
      'payload' => json_encode([
        'message' => "Baris 1\nBaris 2\n\nBaris akhir",
        'phone_number' => 'PHONE_NUMBER_DESTINATION',
        'message_type' => 'text',
        'device_id' => 'YOUR_DEVICE_ID
      ])
```

![PHP Baris Baru](https://i.ibb.co/XFXkKZk/IMG-20210814-195942-edit.png)

**Reply by newfaboc** [reply]
oh. saya lupa harus double quote. terima kasih.

untuk mengirim pesan wa ke beberapa nomor tidak dalam grup bisa kah ?

**Reply by rioastamal** [reply]
Saat ini belum bisa. Fitur ini ada dalam roadmap #11 .
